chainer/functions/activation/lstm.py

```diff
@@ -117,20 +135,25 @@ def backward(self, inputs, grad_outputs):
         if gh is None:
             gh = 0
 
+        a, i, f, o = _extract_gates(x)
         if xp is numpy:
-            co = numpy.tanh(self.c)
+            tanh_a = xp.tanh(a)
+            sig_i = _sigmoid(i)
+            sig_f = _sigmoid(f)
+            sig_o = _sigmoid(o)
```

Comment:
It seems these intermediate values are still stored in the original `LSTM` object. Are they needed? Or is it better to reuse them as before (e.g. by passing them to `__init__`)?

Reply:
Ah, ok. I think it is enough to remove `self.a`. How do you think?

Reply:
It's ok to remove them.